tools/p11ne-db

```diff
@@ -17,6 +17,7 @@ Commands:
     --id            <private-key id>
     --label         <private-key label>
     --key-file      <input PEM file>
+    [--cert-file]   <input certificate PEM file>
```

Comment:
The intent of this change was to simplify testing of Java PKCS11 KeyStore.

During development, we can use this tool to populate both private key and corresponding signed certificate in PKCS11 database, then verify that Java can access the KeyStore alias via PKCS11, which requires both to be present.

Reply:
I just noticed the other point of emphasis in this sentence regarding plain text.
> Plain text certificates should have no place on an instance.

Certificates should be plain text because they represent the public part of the public-private key pair. From a security theory perspective, one should assume certificates are publicly available and already everywhere. Being public, certificates are transmitted in the clear during a TLS handshake, whereas the corresponding private key is heavily protected (encrypted at rest, etc) and never transmitted during a TLS handshake.

Apologies if I misunderstood the intent behind your plain text certificates remark.